telegram/message.py

```diff
@@ -2752,6 +2754,9 @@ def text_html(self) -> str:
         Use this if you want to retrieve the message text with the entities formatted as HTML in
         the same way the original message was formatted.
 
+        Note:
+            |custom_emoji_formatting_note|
```

Comment:
Just curious: Can the `Note:` be part of the replace thingy?

Reply:
I havent found a way, it breaks the indention I think

Reply:
sad :(